diagnostics-app/src/app/app-routing.module.ts

```diff
@@ -10,13 +10,13 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 import { TelemetryDashboardComponent } from './telemetry/telemetry-dashboard/telemetry-dashboard.component';
-import { BatteryComponent } from './diagnostics/battery/battery.component';
```

Comment:
I think we are now deprecating all uses of BatteryComponent since `DiagnosticsDashboardComponent` is a completely new component.

Can we delete `BatteryComponent` from the codebase?

Reply:
Sure. Done.